packages/webapp/webapp_server.js

```diff
@@ -345,8 +345,33 @@ function getBoilerplate(request, arch) {
   return getBoilerplateAsync(request, arch).await();
 }
 
+WebApp.encodeRuntimeConfig = function (rtimeConfig) {
+  return JSON.stringify(encodeURIComponent(JSON.stringify(rtimeConfig)));
+}
+
+WebApp.decodeRuntimeConfig = function (rtimeConfig) {
+  return JSON.parse(decodeURIComponent(JSON.parse(rtimeConfig)));
+}
+
+const runtimeConfig = {
+  'hooks': [],
```

Comment:
We don't need single quotes here. And also not below.

Reply:
got it.  done.